spec/unit/application/lookup_spec.rb

```diff
@@ -145,6 +181,34 @@
       lookup.options[:explain] = true
       lookup.command_line.stubs(:args).returns(['a'])
       expect { lookup.run_command }.to output(<<-EXPLANATION).to_stdout
+Searching for 'lookup_options'
+  Merge strategy hash
+    Data Binding "hiera"
+      No such key: "lookup_options"
+    Data Provider "Hiera Data Provider, version 4"
+      ConfigurationPath "#{environmentpath}/production/hiera.yaml"
+      Data Provider "common"
+        Path "#{environmentpath}/production/data/common.yaml"
+          Original path: common
+          No such key: "lookup_options"
```

Comment:
This just fizzles - there is no real explanation. I would expect something at first level under "Searching for 'lookup options'" to say something like "Result: no lookup_options found, using default merge strategy 'first'" (or something like that).